.github/workflows/docker-validator.yaml

```diff
@@ -2,9 +2,9 @@
 name: Dockerfile and docker-compose Validation
 on:
   push:
-    paths: ['**/Dockerfile', '**/docker-compose.yml']
+    paths: ['Dockerfile', 'docker-compose.yml']
   pull_request:
-    paths: ['**/Dockerfile', '**/docker-compose.yml']
+    paths: ['Dockerfile', 'docker-compose.yml']
```

Comment:
no need to search in the project. In this particular case, we only care about the root level ones